client/my-sites/upgrades/checkout-thank-you/domain-mapping-details.jsx

```diff
@@ -40,24 +49,26 @@ const DomainMappingDetails = ( { domain, registrarSupportUrl } ) => {
 				<li>ns2.wordpress.com</li>
 				<li>ns3.wordpress.com</li>
 			</ul>
-			<p>{ i18n.translate( 'Once you make the change, just wait a few hours and the domain should start loading your site automatically.' ) }</p>
+			<p>{
+				translate(
+					'Once you make the change, just wait a few hours and the domain should start loading your site automatically.'
+				)
+			}</p>
 		</div>
 	);
 
 	return (
 		<div className="checkout-thank-you__domain-mapping-details">
 			<PurchaseDetail
 				icon="cog"
-				title={ i18n.translate( 'Finish setting up your domain' ) }
 				description={ description }
-				buttonText={ i18n.translate( 'Learn more' ) }
+				buttonText={ translate( 'Learn more' ) }
 				href={ supportUrls.MAP_EXISTING_DOMAIN }
 				target="_blank"
 				rel="noopener noreferrer"
-				requiredText={ i18n.translate( 'Almost done! One step remaining…' ) }
 				isRequired />
```

Comment:
Not sure about removing `requiredText`, the blue bar on top is so giant that we might need something to capture their attention.